tools/convert_to_16p2

```diff
@@ -66,19 +63,27 @@ ie. 16-bit yuv planar data samples with the active bits in the high order bits
 
     args = parser.parse_args()
 
-    f = open(args.infilename, "rb")
-
     outfilename = args.infilename + ".16p2"
+    of = open(outfilename, "wb")
 
-    data = bytearray(f.read())
+    word = bytearray(2)
+    temp = bytearray(1)
+    with open(args.infilename, "rb") as fi:
+        with open(args.infilename+'.16p2', "wb") as fo:
+            while True:
+                if fi.readinto(word) == 0:
+                    break
+                if args.bits == 8:
+                    temp = word[1]
+                    word[1] = 0xFF
```

Comment:
I'm sure it doesn't matter since they'll be shifted off later anyway but in the old version of this script don't these bits get set to 0x00, not 0xFF?

Reply:
Yep, I've changed it as per your suggetsion below in aed6f5d